Linux/solution.md

```
@@ -0,0 +1,130 @@
+### TASK1
+
+**Question: 1**  
+
+**Answer**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)